pkgs/misc/emulators/snes9x-gtk/default.nix

```diff
@@ -1,35 +1,38 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, intltool, pkgconfig
-, SDL, zlib, gtk2, libxml2, libXv }:
+{ stdenv, fetchFromGitHub, autoreconfHook, intltool, pkgconfig, SDL2, zlib
+, gtk3, libxml2, libXv, epoxy, minizip, portaudio }:
 
 stdenv.mkDerivation rec {
   name = "snes9x-gtk-${version}";
-  version = "1.54.1";
+  version = "1.56.1";
+  enableParallelBuilding = true;
```

Comment:
Previous indentation / line separation made more sense.

Reply:
Did it? I always found them being scattered like that to be less readable.
I'll change it back.

Reply:
Maybe something like this? https://gist.github.com/yegortimoshenko/f91dae4987dbae252d0112ce1df1e2ad

I think most packages in the tree follow this order because it's one in which things are used: first, `src` is unpacked, then build input PATH is set, then phases are called, etc.